src/graphql/operations/votes.ts

```diff
@@ -20,15 +20,16 @@ async function query(parent, args, context?, info?) {
     app: 'string',
     created: 'number',
     vp: 'number',
-    vp_state: 'string'
+    vp_state: 'string',
+    choice: 'string'
   };
   const whereQuery = buildWhereQuery(fields, 'v', where);
   const queryStr = whereQuery.query;
   const params: any[] = whereQuery.params;
 
   let orderBy = args.orderBy || 'created';
   let orderDirection = args.orderDirection || 'desc';
-  if (!['created', 'vp'].includes(orderBy)) orderBy = 'created';
+  if (!['created', 'vp', 'choice'].includes(orderBy)) orderBy = 'created';
```

Comment:
Why do we need order by choice?

Reply:
> Additional filter options to consider include sorting by ascending voting power, descending voting power, and by choice

https://github.com/snapshot-labs/snapshot/issues/3677#issue-1651211014